### PR TITLE
XW-4547 | Featured Video Attribution

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-4547
@Wikia/x-wing

### CODE REVIEW

File: app/components/article-featured-video.js [hunk 1]
```diff
@@ -23,16 +23,27 @@ export default Component.extend(RenderComponentMixin, JWPlayerMixin, {
...
+	currentVideoDetails: oneWay('initialVideoDetails'),
 	metadata: reads('model.metadata'),
-	placeholderImage: readOnly('model.embed.jsParams.playlist.0.image'),
+	placeholderImage: readOnly('initialVideoDetails.image'),
 
 	placeholderStyle: computed('placeholderImage', function () {
 		return htmlSafe(`background-image: url(${this.get('placeholderImage')})`);
 	}),
 
+	hasAttribution: computed('currentVideoDetails.{username,userUrl,userAvatarUrl}', function () {
```
Comment (hakubo) [inline app/components/article-featured-video.js]: https://emberjs.com/api/ember/2.18/classes/@ember%2Fobject%2Fcomputed/methods/and?anchor=and

**Review by hakubo — CHANGES_REQUESTED** [review]

File: app/components/article-featured-video.js [hunk 2]
```diff
@@ -23,16 +23,27 @@ export default Component.extend(RenderComponentMixin, JWPlayerMixin, {
 
 	classNames: ['article-featured-video'],
 
+	// transparent gif
+	attributionAvatarUrl: 'data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==',
```
Comment (hakubo) [inline app/components/article-featured-video.js]: we use the same gif in about 7 places
what do you think in moving it to
https://github.com/Wikia/mobile-wiki/blob/dev/app/utils/thumbnail.js ?

File: app/components/article-featured-video.js [hunk 3]
```diff
@@ -23,16 +23,27 @@ export default Component.extend(RenderComponentMixin, JWPlayerMixin, {
...
+
 	smartBannerVisible: readOnly('smartBanner.smartBannerVisible'),
 	isFandomAppSmartBannerVisible: readOnly('smartBanner.isFandomAppSmartBannerVisible'),
 
+	initialVideoDetails: readOnly('model.embed.jsParams.playlist.0'),
+	currentVideoDetails: oneWay('initialVideoDetails'),
```
Comment (hakubo) [inline app/components/article-featured-video.js:34]: nice use of oneWay 👍

File: app/templates/components/article-featured-video.hbs [hunk 4]
```diff
@@ -15,3 +15,18 @@
 	</div>
 	<div class="article-featured-video__on-scroll-placeholder"></div>
 </div>
+{{#if hasAttribution}}
+	<div class="article-featured-video__attribution-container {{if hasPortableInfobox
+																																 'has-portable-infobox'}}">
+		<div class="article-featured-video__attribution-inner-container">
```
Comment (hakubo) [inline app/templates/components/article-featured-video.hbs]: is it just github? or something weird happend with spaces here?

**Review by hakubo — APPROVED** [review]

**Review by bkoval — APPROVED** [review]

**Pull request merged**